eng/testing/tests.mobile.targets

```diff
@@ -26,6 +26,12 @@
     <AdditionalXHarnessArguments Condition="'$(ExpectedExitCode)' != ''">--expected-exit-code $(ExpectedExitCode)</AdditionalXHarnessArguments>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetOS)' == 'Android' and '$(EnableTrimming)' == 'true'">
```

Comment:
Could we not just use L12-L23?

Reply:
updated in https://github.com/dotnet/runtime/pull/50510/commits/e5b2dab5dfc79f4e11f80e3cae3d81a568e56457